### PR TITLE
[#9806] Submission page: Fix typo/grammatically incorrect error message

### LINKED ISSUE #9806: Submission page: Fix typo/grammatically incorrect error message
- **Environment**: 
`master` branch at commit 30f56ffea3016decff6026cc46bb03e346ab7ffd

**Steps to reproduce**
1. Fill up a form incorrectly. e.g in MSQ fill up both the "None of the above" option and another option (this is also a bug as seen in #9805)
2. Submit the page

"Some questions do not saved" is shown instead of "Some questions were not saved".

image

### PR DESCRIPTION
**Pull request by erjillsison**
Fixes #9806

### CODE REVIEW

**Comment by erjillsison** [comment]
Ready of initial review

**Comment by Pengyuz** [comment]
LGTM.

File: src/web/app/pages-session/session-submission-page/saving-complete-modal/saving-complete-modal.component.html [hunk 1]
```diff
@@ -14,7 +14,7 @@ <h5 class="modal-title">
 </div>
 <div class="modal-body">
   <p *ngIf="notYetAnsweredQuestions.length !== 0">
-    <span class="unanswered-exclamation-mark"> &#10071; </span> Note that some questions are yet to be answered.
```
Comment (xpdavid) [inline src/web/app/pages-session/session-submission-page/saving-complete-modal/saving-complete-modal.component.html]: Can the CSS class also be deleted?

Comment (erjillsison) [reply src/web/app/pages-session/session-submission-page/saving-complete-modal/saving-complete-modal.component.html]: Updated

**Pull request merged**